#x,y,z,qx,qy,qz,qw
gp_T_hand:     [-0.01,  -0.21,  0.0,   0.0,     0.7071,  0.0,     0.7071]
gp_T_palm:     [ 0.0,    0.0,   0.0,   0.0,     0.0,     0.0,     1.0   ]
hand_T_palm:   [ 0.0,    0.21,  0.01,  0.0,    -0.7071,  0.0,     0.7071]
hand_T_marker: [ 0.0,    0.21,  0.01,  0.0,    -0.7071,  0.0,     0.7071]
